gradient/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p input
+if [ ! -e "input/UK-dem-50m-4326.tif" ]; then
+	wget https://play.abstreet.org/dev/data/input/shared/elevation/UK-dem-50m-4326.tif.gz -P input
+	gunzip input/UK-dem-50m-4326.tif.gz
+fi
+
+mkdir -p tmp
+# Assume the root directory has the osm.pbf, used by many other scripts in this repo
+time cargo run --release ../england-latest.osm.pbf input/UK-dem-50m-4326.tif
+
+time tippecanoe tmp/gradient.geojson \
```

Comment:
This layer is particularly meaningless at low zooms. I've been fighting with tippecanoe to set a minimum zoom, but no success yet. Once I figure out all the tricks, this and many other layers (like road width, speed, cycle paths, etc) can be simplified.